### ISSUE TITLE
PTN001679972 (PTHR43917)

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#4458]
* **PTHR ID & PTN node:**
PTN001679972 (PTHR43917)

* **Sequences with problematic annotation (ID + gene/protein name):**
Fly GstT4 (FBgn0030484/Q8MRM0)

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**
erroneous propagation

Fly GstT4 has a manual "NOT enables glutathione transferase activity" annotation, but is getting a positive annotation to the same term from PTN001679972

**Comment by marcfeuermann** [comment]
Yes, this is a recurrent problem leading from a false good idea in my opinion. Corrected.
